_docs/_latest/concepts/models.md

```diff
@@ -1,84 +1,77 @@
 ---
 title: Models
-order: 100
+order: 003
 ---
 
 # {{ page.title }}
 
 Starting with version 2.0, Forseti introduces the use of data models.
 
 The data model is an additional pool of data that is relational in nature,
-and is created from the flat json data in inventory, that's captured from GCP
-API responses. With the relational data, we can more easily see the
-relationships and the inheritances between resources, as well as being able
-to query for them.  One example is to see all the accesses that a service
-account may have, across all the projects.
+and is created from the flat json data in inventory. With the relational data, Forseti 
+can more easily understand the entire relationship, including inheritance between resources. Models
+allow for easier querying against the entire computed policy.
 
 Another important concept to keep in mind is that the data models are not meant
-to be persistent. The data models need to be created before use, and then
-destroyed after use. So, before you want to use Explain, you will need to
-create your own set of data model, and then delete them after finish using
-Explain.
+to be persistent. Before using Scanner or Explain a data model must be created. Once you're done 
+using the model it should be deleted.
 
-Also important, the Scanner and Explainer components depend on the data models
-as the data source, i.e. it is a pre-requiste for the data models to be
-created successfully, before the Scanner or the Explainer can be used.
+Both Scanner and Explain depend on the data models being present, therefore creating a valid data 
+model prior to using Scanner or Explain is **required**.
 
-## Tables
+---
 
-Data models are stored in its own set of tables, which are named
-with the `<model_handle>_<table_name>` and are tied to each other by specific
-relationships.  At any given time, multiple set of tables can exist,
-either created by the cron job, or by other users. The table sets are
-listed in the `models` table.  
+## How data models are stored
 
-### binding_members
+Data models are stored in its own set of tables, which are named with the
+`<model_handle>_<table_name>` and are tied to each other by specific relationships.  At any given
+time, multiple set of tables can exist, either created by the cron job, or by other users. The
+table sets are listed in the `models` table.  
 
-This table is a join table that connects `members` table with the `bindings`
-table, thus making it possible to know what resources each member can access.
+### The `binding_members` table
 
-### bindings
+This table is a join table that connects `members` table with the `bindings` table, thus making it
```

Comment:
Simplify: "so you can know what resources"

Reply:
done